packages/checkbox/src/checkbox.vue

```diff
@@ -113,7 +113,9 @@ export default defineComponent({
     const instance = getCurrentInstance()
 
     onMounted(() => {
-      instance.vnode.el.setAttribute('aria-controls', props.controls)
+      if (props.indeterminate) {
+        instance.vnode.el.setAttribute('aria-controls', props.controls)
+      }
     })
```

Comment:
Can we set this via template instead of raw DOM manipulation? In my understanding that this property is meant to set on `label` component

Reply:
Reference: https://www.w3.org/TR/wai-aria-1.1/#aria-controls

Reply:
YES.